src/lib/containers/DirectDownload.tsx

```diff
@@ -63,8 +63,9 @@ const DirectDownload: React.FunctionComponent<DirectFileDownloadProps> = props =
 
     if (!preSignedURL) {
       console.log('Fail to get file download link')
+    } else {
+      window.open(preSignedURL)
```

Comment:
But only open the presigned URL in a new window if we were able to get a presigned URL